src/controllers/trainee/routes.ts

```diff
@@ -0,0 +1,33 @@
+import * as express from 'express';
+import { authMiddleWare, validationHandler } from '../../libs';
+import { traineeRoutes } from './Controller';
+import { default as validation } from './validation';
+
+const traineeRouter = express.Router();
+
+traineeRouter.route('/')
+
+    .get(
+        authMiddleWare('getUsers', 'write'),
```

Comment:
'write' should be 'read' over here

Reply:
If I use read permission then the trainee also gets access to info of all trainees.
So, the logic here is that only people who have write access can actually read trainee details